docs/src/dev/developer/development-environment.asciidoc

```diff
@@ -497,6 +503,43 @@ docker/gremlin-server.sh 3.4.2
 To be a bit more clear, the version can not be a Docker tag like "latest" because there is no such TinkerPop artifact
 that has been published with that version number.
 
+[[docker-testing]]
+== Testing with Docker
+
+Some GLV allow you to run tests without setting up Java and without building of other parts of the server. Currently
```

Comment:
Is it true that one doesn't need to set up Java? Because in the instructions below we are telling people to build the gremlin server, which needs Java?

Reply:
Can you run gremlin-go test without building server?

Reply:
But running anything in Maven needs a Java JDK though?

Reply:
I'd say it's more true that it doesn't need to build server, but Java is still required. And it is specified as an requirement for TinkerPop anyways.

Reply:
yes, but `run.sh` don't use Maven

Reply:
or manual `docker-compose up`

Reply:
fixed